Externals.cfg

```diff
@@ -1,9 +1,9 @@
 [regional_workflow]
 protocol = git
-repo_url = https://github.com/NOAA-EMC/regional_workflow
```

Comment:
@chan-hoo, Thanks to you and @BenjaminBlake-NOAA for your work on these WCOSS mods!  We still want to point to the authoritative repo in the Externals.cfg file, so once [this PR](https://github.com/NOAA-EMC/regional_workflow/pull/385) goes into regional_workflow, can you modify the Externals.cfg in this PR to make sure it points to the authoritative regional_workflow and to the latest hash?  Thanks!

Reply:
@JeffBeck-NOAA , Sure. Once the PR (https://github.com/NOAA-EMC/regional_workflow/pull/385) is merged, I'll modify this and test this PR with the authoritative repo again.

Reply:
@JeffBeck-NOAA, I've change the file to the original.

Reply:
Hi @chan-hoo, can you have it point to the latest hash of regional_workflow (the one for your WCOSS PR)?  Then this will serve as the stable hash for you, @BenjaminBlake-NOAA, @RatkoVasic-NOAA, and everyone at EMC to use in the future.  Thanks!

Reply:
@JeffBeck-NOAA, I've done it.

Reply:
@chan-hoo, Thanks!